include/i2c.h

```diff
@@ -45,8 +45,9 @@ extern "C" {
 /** I2C Ultra Fast Speed */
 #define I2C_SPEED_ULTRA			(0x5)
 
-/** @cond INTERNAL_HIDDEN */
 #define I2C_SPEED_SHIFT			(1)
+#define I2C_SPEED_SET(speed)		((speed) << I2C_SPEED_SHIFT)
+/** @cond INTERNAL_HIDDEN */
```

Comment:
Maybe we should remove `@cond INTERNAL_HIDDEN`? If _SET macro is visible/documented then _GET should be too.